protocol/v2/ssv/spectest/sync_committee_aggregator_proof_type.go

```diff
@@ -73,13 +73,13 @@ func overrideStateComparisonForSyncCommitteeAggregatorProofSpecTest(t *testing.T
 }
 
 func keySetForShare(share *spectypes.Share) *testingutils.TestKeySet {
-	if share.Quorum == 5 {
+	if len(share.Committee) == 5 {
 		return testingutils.Testing7SharesSet()
 	}
-	if share.Quorum == 7 {
+	if len(share.Committee) == 7 {
 		return testingutils.Testing10SharesSet()
 	}
-	if share.Quorum == 9 {
```

Comment:
Here the length of the Committee is compared to 5, 7 and 9. But these are actually quorum sizes. In spec, we have:
```go
func KeySetForShare(share *types.Share) *TestKeySet {
	if len(share.Committee) == 7 {
		return Testing7SharesSet()
	}
	if len(share.Committee) == 10 {
		return Testing10SharesSet()
	}
	if len(share.Committee) == 13 {
		return Testing13SharesSet()
	}
	return Testing4SharesSet()
}
```